packages/fx-core/src/plugins/solution/fx-solution/v2/grantPermission.ts

```diff
@@ -184,11 +189,19 @@ async function grantPermissionImpl(
         ui?.showMessage("info", message, false);
       }
 
-      ui?.showMessage(
-        "info",
-        `\nSkip grant permission for Azure resources. You may want to handle that via Azure portal. `,
-        false
-      );
+      if (CollaborationUtil.isSpfxProject(param.ctx)) {
+        ui?.showMessage(
+          "info",
+          `\nIf added user is not SharePoint site admin, you need to handle that via SharePoint admin center.`,
```

Comment:
More specific: is not a SharePoint App Catalog site admin

Reply:
Here is the doc for manage admins, maybe you can have a Readmore here.

https://docs.microsoft.com/en-us/sharepoint/manage-site-collection-administrators

Reply:
Thank you, will update